heed/src/database.rs

```diff
@@ -71,12 +73,23 @@ impl<'e> DatabaseOpenOptions<'e, Unspecified, Unspecified> {
     }
 }
 
-impl<'e, KC, DC> DatabaseOpenOptions<'e, KC, DC> {
+impl<'e, KC, DC, C> DatabaseOpenOptions<'e, KC, DC, C> {
     /// Change the type of the database.
     ///
     /// The default types are [`Unspecified`] and require a call to [`Database::remap_types`]
     /// to use the [`Database`].
-    pub fn types<NKC, NDC>(self) -> DatabaseOpenOptions<'e, NKC, NDC> {
+    pub fn types<NKC, NDC>(self) -> DatabaseOpenOptions<'e, NKC, NDC, NoopComparator> {
+        DatabaseOpenOptions {
+            env: self.env,
+            types: Default::default(),
+            name: self.name,
+            flags: self.flags,
+        }
+    }
+    /// Change the customized key compare function of the database.
+    ///
+    /// By default no customized compare function will be set when opening a database.
+    pub fn comparator<NC>(self) -> DatabaseOpenOptions<'e, KC, DC, NC> {
```

Comment:
Should we rename this method into `key_comparator`, what do you think?

Reply:
Great idea, and code is updated!